src/Testable/Internal.elm

```diff
@@ -11,7 +11,7 @@ type Cmd msg
 
 
 type Task error success
-    = HttpTask Http.Request (Result Http.RawError Http.Response -> TaskResult error success)
+    = HttpTask Http.Request Http.Settings (Result Http.RawError Http.Response -> TaskResult error success)
```

Comment:
Can you flip the order so that Settings is before Request, to match the order in Http.send